docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -329,11 +413,13 @@
     "    SafetySetting,\n",
     ")\n",
     "\n",
-    "index = GoogleIndex.from_corpus(corpus_id=\"123\")\n",
+    "index = GoogleIndex.from_corpus(corpus_id=corpus_id(1))\n",
```

Comment:
this will always throw an error, should we use the corpus_id in the Basic Usage here?

Reply:
You may be reading an old version of the code. The current code uses `corpus_id(1)` for the `corpus_id` argument.

Reply:
Fixed as stated in the previous comment.